From 2725785480df3de2f86956d668be95a46b9f23f6 Mon Sep 17 00:00:00 2001
From: Danny Baumann <dannybaumann@web.de>
Date: Mon, 2 Dec 2019 09:34:22 +0100
Subject: [PATCH 15/19] Beautify call stats details category headers.

Make them consistent with how pref category headers are displayed.

Change-Id: I4ee8bb2dae11970051552d14df2d58d7ab6a3fdf
---
 .../dialer/callstats/res/layout/call_stats_detail_info.xml | 6 +++---
 java/com/android/dialer/callstats/res/values/styles.xml    | 7 +++++++
 2 files changed, 10 insertions(+), 3 deletions(-)

diff --git a/java/com/android/dialer/callstats/res/layout/call_stats_detail_info.xml b/java/com/android/dialer/callstats/res/layout/call_stats_detail_info.xml
index 810a5bf45..4f4e59f02 100644
--- a/java/com/android/dialer/callstats/res/layout/call_stats_detail_info.xml
+++ b/java/com/android/dialer/callstats/res/layout/call_stats_detail_info.xml
@@ -45,7 +45,7 @@
       android:text="@string/call_stats_title_durations"
       android:layout_marginBottom="@dimen/call_log_vertical_padding"
       android:textColor="?android:attr/colorAccent"
-      style="@android:style/TextAppearance.Material.Subhead" />
+      style="@style/TextAppearance.CallStatsCategoryTitle" />
 
     <TextView
       android:id="@+id/total_duration_total"
@@ -120,7 +120,7 @@
     android:layout_marginBottom="@dimen/call_log_vertical_padding"
     android:text="@string/call_stats_title_count"
     android:textColor="?android:attr/colorAccent"
-    style="@android:style/TextAppearance.Material.Subhead" />
+    style="@style/TextAppearance.CallStatsCategoryTitle" />
 
   <TextView
     android:id="@+id/total_count_total"
@@ -209,7 +209,7 @@
     android:layout_marginBottom="@dimen/call_log_vertical_padding"
     android:text="@string/call_stats_title_average_duration"
     android:textColor="?android:attr/colorAccent"
-    style="@android:style/TextAppearance.Material.Subhead" />
+    style="@style/TextAppearance.CallStatsCategoryTitle" />
 
   <include
     android:id="@+id/in_average"
diff --git a/java/com/android/dialer/callstats/res/values/styles.xml b/java/com/android/dialer/callstats/res/values/styles.xml
index 7a4bc453d..3aae79757 100644
--- a/java/com/android/dialer/callstats/res/values/styles.xml
+++ b/java/com/android/dialer/callstats/res/values/styles.xml
@@ -26,4 +26,11 @@
     <item name="redColor">@color/missed_call</item>
     <item name="orangeColor">@color/blocked_call</item>
   </style>
+
+  <style name="TextAppearance.CallStatsCategoryTitle" parent="TextAppearance.AppCompat.Body2">
+    <item name="android:textAllCaps">true</item>
+    <item name="android:textSize">11sp</item>
+    <!-- 0.8 Spacing, 0.8/11 = 0.072727273 -->
+    <item name="android:letterSpacing">0.072727273</item>
+  </style>
 </resources>
-- 
2.17.1

